apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -122,6 +123,18 @@ type TempoStackSpec struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Observability"
 	Observability ObservabilitySpec `json:"observability,omitempty"`
+
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Extra Configurations"
+	ExtraConfig *ExtraConfigSpec `json:"extraConfig,omitempty"`
+}
+
+// ExtraConfigSpec defines extra configurations for tempo that will be merged with the operator generated, configurations defined here
+// has precedence and could override generated config.
+type ExtraConfigSpec struct {
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tempo Extra Configurations"
+	Tempo *apiextensionsv1.JSON `json:"tempo,omitempty"`
```

Comment:
```suggestion
	Tempo apiextensionsv1.JSON `json:"tempo,omitempty"`
```

Reply:
I was thinking on put it as a pointer, just in case we wanted to detect the absent of that field. make sense?

Reply:
`len(tempo.Spec.ExtraConfig.Tempo.Raw) == 0` should work to detect the absence of the field

Reply:
Make sense! I changed it.